kymatio/scattering1d/convolution.py

```diff
@@ -0,0 +1,27 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+class Conv1dFFT(nn.Module):
```

Comment:
We capitalize `D` elsewhere, right? On the other hand, we may write FFT as `Fft` to make it easier to read.